modules/ROOT/pages/cache.adoc

```diff
@@ -0,0 +1,25 @@
+= Optionally Configure the Cache
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager can improve performance with a platform cache.
+
+The platform cache has two parts: a session cache that holds data specific to one user and their session, and an organization cache that holds data that will be accessed by multiple users in the organization.
+
+If you wish to change the default configuration for the caches:
+
+. Log in to your Salesforce organization.
+. In *Setup*, click *Custom Metadata Types*.
+. Next to the label *ACM Platform Cache Metadata*, click *Manage Records*.
+. Next to the label *ACM Configuration*, click *Edit*.
+. Use the *OrgCacheEnabled* and *SessionCacheEnabled* checkboxes to enable or disable the caches.
+. Set the number of seconds that cached data remains in the cache for the API documentation, API instance, and API query entities with the *ApiDocumentationLifetime*, *ApiInstancesLifetime*, and *ApiQueryLifetime* fields.
```

Comment:
Set the number of seconds that cached data remains in the cache for API versions, documentation content, instances, and queries with the *API Versions Lifetime*,  *API Documentation Lifetime*, *API Instances Lifetime*, and *API Query Lifetime* fields.

Reply:
Fixed.